rules/windows/credential_access_posh_request_ticket.toml

```diff
@@ -14,6 +14,41 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Kerberos Ticket Request"
+note = """## Triage and analysis
+
+### Investigating MS Office Macro Security Registry Modifications
```

Comment:
```suggestion
### Investigating Explicit PowerShell Kerberos Ticket Request
```
Current header indicates investigating activity related to Microsoft Office Macro's that when executed, modify the registry.

Reply:
Nice catch! 👍🏼